Notesnook.API/Models/SyncItem.cs

```diff
@@ -111,7 +111,7 @@ public double Version
         public string Algorithm
         {
             get; set;
-        } = Algorithms.Default;
+        }
```

Comment:
Let's revert this for now.

Reply:
Because of the default value, it always sets the incoming request's algorithm to xcha-argon2id-7, regardless of what algorithm is set in the request payload. This isn't an issue because that is the algorithm we're using for everywhere now. Just wanted to highlight this.